### ISSUE TITLE
Add a `license` field

### ISSUE THREAD

**Issue by jvoisin** [saimn/sigal#176]
Currently, there is only an `author` field in the configuration file, it would be nice to also have a `license` one.

**Comment by saimn** [comment]
You can add any field in the config file and access it in the template. So what is needed is to integrate this in the themes (if the license is present).

**Comment by jvoisin** [comment]
This was implied ;)

**Comment by saimn** [comment]
Yes, just thought it was better to clarify the issue in case someone wants to take on this.
